source/Planet.cpp

```diff
@@ -408,6 +428,22 @@ const Sale<Outfit> &Planet::Outfitter() const
 
 
 
+// Get the outfitter description
+string Planet::OutfitterDescription() const
+{
+	if(!outfitterDescription.empty())
+		return outfitterDescription;
+
+	string outfitterDescriptionFromSales = "";
```

Comment:
same here
also maybe the outfitter and shipyard description should add up to each others.
Furthermore, this kind of info could be shown from the MapDetailPanel when the relevant thing is selected on the planet card? (like when the outfitter/shipyard is selected)